src/System.Reflection.Metadata.Cil/src/Decoder/CilTypeProvider.cs

```diff
@@ -229,9 +237,10 @@ public CilType GetTypeFromReference(TypeReferenceHandle handle, bool? isValueTyp
             return type;
         }
 
+        [CLSCompliant(false)]
         public string GetParameterList(MethodSignature<CilType> signature, ParameterHandleCollection? parameters = null)
         {
-            ImmutableArray<CilType> types = signature.ParameterTypes;
+            var types = signature.ParameterTypes;
```

Comment:
Why the change to var? Now without going to definition I do not know what type these are.